proxy_docker/app/script/confirmation.sh

```diff
@@ -7,44 +7,35 @@
 . ./responsetoclient.sh
 . ./computefees.sh
 . ./blockchainrpc.sh
+. ./watchrequest.sh
 
-confirmation_request()
-{
-  # We are receiving a HTTP request, let's find the TXID from it
-
-  trace "Entering confirmation_request()..."
-
-  local request=${1}
-  local txid=$(echo "${request}" | cut -d ' ' -f2 | cut -d '/' -f3)
-
-  confirmation "${txid}"
-  return $?
-}
-
+# Expecting 2 params
+#
+# 1: base64 encoded {"amount": 0.00000000, "confirmations": 1,"generated": true,"blockhash": "67862b2428039f1e4e088ae74e44230705931784862c4f03f8978d6d53adad83",...}
+# 2: boolean bypass_callbacks (optional)
+#
 confirmation() {
   (
   flock -x 201
 
   trace "Entering confirmation()..."
 
-  local returncode
-  local txid=${1}
+  local tx_details=$(echo ${1} | base64 -d)
```

Comment:
Ok, we got `tx_details` needed to have wallet-specific information about the tx and we got `tx_raw_details` needed to get some other information not found in `gettransaction` but found in `getrawtransaction`.

Here, we are receiving `tx_details` (result of `gettransaction`) but a few lines below we are calling `getrawtransaction` to get the rest of the details (into `tx_raw_details`).

I would like that we call `getrawtransaction` on the Bitcoin Core container's side in the walletnotify script and include the result in the published message.  So we would have something like `{"gettransaction":{...},"getrawtransaction":{...}}` in the base64 message.

Reply:
...instead of calling get_rawtransaction from the proxy.

Reply:
Ohhh wait a minute.  We can use `gettransaction txid true true` on the node to get everything we need, including the rawtransaction info (in the `decoded` field)!  Let's use it in walletnotify.sh and publish the result as message!

Reply:
NIce, I did not see the third argument on chainquery.com, it's not up to date (second time !!).  I'll add this !!